src/mission.h

```diff
@@ -187,7 +185,7 @@ struct mission_type {
     int item_count = 1;
     npc_class_id recruit_class = npc_class_id( "NC_NONE" );  // The type of NPC you are to recruit
     int target_npc_id = -1;
-    std::string monster_type = "mon_null";
+    mtype_id monster_type = mtype_id( "mon_null" );
```

Comment:
Please use `mtype_id::NULL_ID()` instead of the string literal, here and at all the other places that use a string literal.